Userland/Libraries/LibJIT/GDB-unsupported.cpp

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2023, Jesús Lapastora <cyber.gsuscode@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibJIT/GDB.h>
+
+namespace JIT::GDB {
+Optional<FixedArray<u8>> build_gdb_image(ReadonlyBytes /*code*/, StringView /*file_symbol_name*/, StringView /*code_symbol_name*/)
```

Comment:
Omitting the parameter names entirely should be fine here, as this is a stub implementation and the names are already in the header file.